### ISSUE TITLE
utils.py TypeError: unhashable type: 'set'

### ISSUE THREAD

**Issue by hartsantler** [godotengine/godot-blender-exporter#251]
**OS:**
linux64 fedora

**Godot version:**
godot stock fedora

**Blender version:**
blender2.8 official

**Issue description:**
```
[INFO]: Exporting Blender object: Cone.002
[INFO]: Exporting Blender object: cube.005
Error: Traceback (most recent call last):
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/__init__.py", line 198, in execute
    return export_godot.save(self, context, **keywords)
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/export_godot.py", line 318, in save
    exp.export()
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/export_godot.py", line 267, in export
    self.export_scene()
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/export_godot.py", line 211, in export_scene
    self.export_object(obj, root_gd_node)
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/export_godot.py", line 153, in export_object
    self.export_object(child, exported_node)
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/export_godot.py", line 123, in export_object
    parent_gd_node)
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/converters/mesh.py", line 45, in export_mesh_node
    mesh_id = mesh_exporter.export_mesh(escn_file, export_settings)
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/converters/mesh.py", line 159, in export_mesh
    key = MeshResourceKey('ArrayMesh', self.object, export_settings)
  File "/home/raptor/Downloads/blender-2.80/2.80/scripts/addons/io_scene_godot/converters/utils.py", line 81, in __init__
    self._hash = hash(self._data)
TypeError: unhashable type: 'set'
```

**Comment by hartsantler** [comment]
seems to be caused by objects with a boolean modifier on it

**Comment by Jason0214** [comment]
looks like same issue with #247

**Comment by Jason0214** [comment]
Should be fixed by #262
